src/Microsoft.AspNet.Mvc.Core/DefaultValidationExcludeFiltersProvider.cs

```diff
@@ -17,13 +16,12 @@ public class DefaultValidationExcludeFiltersProvider
         /// Initializes a new instance of the DefaultBodyValidationExcludeFiltersProvider class.
         /// </summary>
         /// <param name="options">An accessor to the <see cref="MvcOptions"/> configured for this application.</param>
-        /// <param name="typeActivator">An <see cref="ITypeActivator"/> instance used to instantiate types.</param>
```

Comment:
Docs for all of these need to be updated, not just have `typeActivator` taken out